src/main/java/creativei/entity/FollowUp.java

```diff
@@ -20,7 +20,7 @@
 @Table(name = "Follow_Up")
 public class FollowUp extends BaseEntity implements Serializable {
 
-    @ManyToOne(cascade = CascadeType.MERGE,fetch = FetchType.LAZY)
+    @ManyToOne(cascade = CascadeType.REFRESH,fetch = FetchType.LAZY)
```

Comment:
Why do you need to use REFRESH on inquiry?

Reply:
because on updating if MERGE is there it is updating the inquiry entity too with the inquiryId we had provided

Reply:
How is REFRESH helping this? Can we avoid?

Reply:
there is no need of cascade type so i removed it in commit 3f3804c